apps/molgenis-components/src/components/forms/formUtils/formUtils.ts

```diff
@@ -293,8 +296,8 @@ function isInvalidHyperlink(value: any) {
   return value && !HYPERLINK_REGEX.test(value);
 }
 
-function containsInvalidHyperlink(hyperlinks: any) {
-  return hyperlinks.some((hyperlink: string) => isInvalidHyperlink(hyperlink));
+function containsInvalidHyperlink(hyperlinks: string[] | undefined) {
+  return !!hyperlinks?.some(isInvalidHyperlink);
```

Comment:
`!!` casts hyperlinks to a boolean, so this will always return false, since there is no .some for bools